<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="managerTabs">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="generalTab">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>761</width>
          <height>491</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="generalTabLayout"/>
       </widget>
      </widget>
      <widget class="QWidget" name="employeeTab">
       <attribute name="title">
        <string>Employee Manager</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>761</width>
          <height>491</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="employeeTabLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Employees</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="employeeList"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="orderTab">
       <attribute name="title">
        <string>Order Manager</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>761</width>
          <height>491</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="orderTabLayout"/>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConfigure"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
